src/Sylius/Bundle/ThemeBundle/Command/AssetsInstallCommand.php

```diff
@@ -34,7 +34,7 @@ protected function configure(): void
         $this
             ->setName('sylius:theme:assets:install')
             ->setDefinition([
-                new InputArgument('target', InputArgument::OPTIONAL, 'The target directory', 'web'),
+                new InputArgument('target', InputArgument::OPTIONAL, 'The target directory', 'public'),
```

Comment:
This change should be reverted - it's a BC break (it is still possible to use Sylius 1.3 with old directory structure).

Reply:
I've reverted the default value change.